tests/Integration/Defer/DeferTest.php

```diff
@@ -303,7 +303,7 @@ public function itCancelsDefermentAfterMaxExecutionTime(): void
         }
         ";
 
-        /** @var \Nuwave\Lighthouse\Defer\Defer $defer */
+        /* @var \Nuwave\Lighthouse\Defer\Defer $defer */
```

Comment:
Docblocks start with `/**`

Reply:
http://docs.phpdoc.org/guides/docblocks.html#anatomy-of-a-docblock

Reply:
When they are multiline, not one a single line

Reply:
I can change that though

Reply:
Did you look at the link?

Reply:
Yes, but I've been tricked by `createNodeResolver ` 😞

Reply:
Done : https://github.com/nuwave/lighthouse/pull/664/commits/8ec24bb632726e667920144067c6896e48604825

Reply:
Seems this still breaks StyleCI: https://github.styleci.io/analyses/z9j7R6?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link